Package.swift

```diff
@@ -30,6 +30,18 @@ let package = Package(
             name: "WorkflowCombine",
```

Comment:
Does this also need to be added to CI, or will it run already?

Reply:
Good question. I'm not familiar with the Github CI infra perhaps @mjohnson12 you have a better idea?

Reply:
I believe the CI build using the podspec for running the tests which doesn't including anything that requires iOS 13+ since cocoapods would require workflow to require iOS 13+ so those are commented out in Development.podspec.

There is one CI test that runs with package manager but I don't know which tests it runs.

Reply:
Looking at the output of the swift package manager CI step and it does not run the WorkflowCombine tests.

Reply:
I've pushed up an update that adds WorkflowCombine to the tests that run for the SPM CI build step.